pkg/metadata/client.go

```diff
@@ -39,3 +40,35 @@ func NewStore(path string) (Store, error) {
 func (b boltClient) Close() error {
 	return b.db.Close()
 }
+
+// TODO: Verify libvirt domain update info or callbacks status before reset the resource update in progress flag
+//       if libvirt is still updating it, the don't reset it
```

Comment:
How do we know if libvirt is still updating?  Can you please file an issue so we can track it?

Reply:
will need some research here in libvirt. the callbacks in libvirt current don't seem offer this status check. so this will be the best effort approach, i.e. if there is no such functionality, we will force reset it. our logic in the CRI should skip this if the resource level is the same. so this is just for further optimization.
issue #56 is to track this todo.